.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build-Publish
+
+on:
```

Comment:
hva betyr dette?

Reply:
altså det som kommer under on

Reply:
>   push:
>     branches:
>       - '**'
Bygger alltid on push uavhengig av branch man pusher til
>   schedule:
>     - cron: '22 22 * * 3'
Bygger hver uka kl 22.22 på onsdager - det er til å få med ubuntu oppdateringer fra baseimage som bygges kl 22.22 på tirsdager :)
>   workflow_dispatch:
Man kan trigge bygget manuelt også
<img width="297" alt="image" src="https://user-images.githubusercontent.com/5756550/216299336-1b4ecdee-4318-49f6-81dc-1a720d35ad14.png">

Reply:
takk for godt svar!